client/src/pages/FlightData/index.js

```diff
@@ -77,7 +79,9 @@ const FlightData = () => {
 		defaultAlt: defaultAlt,
 		currentDistance: currentDistance,
 		firstJump: firstJump,
-		firstPoint: firstPoint
+		firstPoint: firstPoint,
+		planePoints: planePoints,
+		Awaypoint: Awaypoint
```

Comment:
Better name? What's the purpose?

Reply:
Nice work with these getters and setters. However, I'd call `Awaypoint` something different - maybe `toNextWaypoint`?